src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -38,4 +38,20 @@
       </attribute>
     </additionalAttributes>
   </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Formula.Evaluate</oldName>
+    <newName>DSCore.Math.EvaluateFormula</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.Xor</oldName>
+    <newName>DSCore.Math.Xor</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.And</oldName>
+    <newName>DSCore.Math.And</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.Or</oldName>
+    <newName>DSCore.Math.Or</newName>
```

Comment:
Ah, there it is:
https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodeModels/Logic/AndOr.cs#L80